Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -525,31 +525,46 @@ extension LazyTensorOperation: TFTensorOperation {
     }
 }
 
-extension LazyTensorOperation.Attribute: CustomStringConvertible {
-    var description: String {
-        switch self {
-        case .BoolValue(let v): return "\(v)"
-        case .IntValue(let v): return "Int(\(v))"
-        case .FloatValue(let v): return "Float(\(v))"
-        case .DoubleValue(let v): return "Double(\(v))"
-        case .StringValue(let v): return "\"\(v)\""
-        case .BoolArray(let values): return arrayAsString("", values)
-        case .IntArray(let values): return arrayAsString("Int", values)
-        case .FloatArray(let values): return arrayAsString("Float", values)
-        case .DoubleArray(let values): return arrayAsString("Double", values)
-        case .StringArray(let values): return arrayAsString("String", values)
-        case .ConstTensor(let v): return "Const(\(v))"
-        case .TensorDataTypeValue(let v): return dataTypeAsString(v)
+extension TFETensorHandle: CustomStringConvertible {
+    public var description: String {
+        let dtype = TFE_TensorHandleDataType(self._cTensorHandle)
```

Comment:
I think this switch-on-type-and make Tensor / TensorHandle is already somewhere else. Hopefully those would be consolidated to exist only once.